lib/Common/Core/Output.cpp

```diff
@@ -540,9 +548,14 @@ Output::CaptureEnd()
 {
     Assert(s_capture);
     s_capture = false;
+#ifdef _WIN32
     bufferFreeSize = 0;
     bufferAllocSize = 0;
     char16 * returnBuffer = buffer;
     buffer = nullptr;
+#else
+    char16 * returnBuffer = nullptr;
```

Comment:
This changes the behavior when `CHECK_MEMORY_LEAK` is defined, since there is code there that assumes it is writing to a buffer, and stores that buffer elsewhere.

I suppose that functionality isn't necessary on xplat, at least for now? It seems that it shouldn't break anything, just change the behavior of the debug logging.

Reply:
I'm not sure if that functionality even works on xplat; we'll likely use ASAN moving forward for xplat memory leak checking in similar scenarios regardless.